ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/ipsec/fips/provision/cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec-fips-provision
+  steps:
+  - chain: ipi-conf-gcp
+  - ref: ovn-conf 
+  - ref: ovn-conf-ipsec-manifest
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  env:
+  - name: FIPS_ENABLED
+    default: "true"
+  documentation: |-
+    Create an fips enabled IPI cluster with OVN and ipsec for QE e2e tests.
```

Comment:
For fips, there is no need to add a new chain/workflow now.
You can add env `FIPS_ENABLED: "true" in the config directly for an existing chain/workflow.
```
- as: aws-c2s-ipi-disconnected-private-fips-p2-f14
  cron: 32 14 11,26 * *
  remote_api: true
  steps:
    cluster_profile: aws-c2s-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
      FIPS_ENABLED: "true"
    test:
    - chain: cucushift-installer-check-cluster-health
    workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
```

Reply:
yes, I just add one more fips workflow for cluster-bot command line workflow-launch. Currently, we just have one fips workflow available.

Reply:
Detail see, https://github.com/openshift/release/blob/master/core-services/ci-chat-bot/workflows-config.yaml

Reply:
Workflow does not need to specify fips in it now. E.g, for a default non-fips workflow, fips can be enabled by adding env `FIPS_ENABLED=true`

Reply:
That's okay. Closing this PR.